src/gui/qgscollapsiblegroupbox.h

```diff
@@ -130,6 +130,7 @@ class GUI_EXPORT QgsCollapsibleGroupBoxBasic : public QGroupBox
     void checkToggled( bool ckd );
     void checkClicked( bool ckd );
     void toggleCollapsed();
+    void setStyleSheet( const QString &style );
```

Comment:
Just add docs, don't expand the missing doc list!

Reply:
I followed the same as the other overloaded slots because the documentation I suppose is based on base class documentation. BTW I'll add doc.

Reply:
added